src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/CaseDataContent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@Builder
+public class CaseDataContent {
```

Comment:
Something like `CaseEvenCreationPayload` would be a more precise answer. Case data content is actually the data field inside.

Reply:
Fixed in https://github.com/hmcts/aac-manage-case-assignment/pull/150/commits/04ca3c6315a335ded9f9683f351c1dbf78f7d120